descriptors/ModuleDescriptor-template.json

```diff
@@ -263,7 +266,13 @@
         {
           "methods": ["POST"],
           "pathPattern": "/inn-reach/central-servers/{centralServerId}/contributions",
-          "permissionsRequired": ["inn-reach.contributions.item.post"]
+          "permissionsRequired": [
+            "inn-reach.contributions.item.post",
+            "circulation.requests.collection.get",
+            "inventory-storage.instance.iteration.post",
+            "inventory-storage.inventory-view.instances.collection.get",
+            "inventory-storage.material-types.collection.get"
+          ]
```

Comment:
it should require only `inn-reach.contributions.item.post` all the others should be added as `modulePermissions`